pkg/cmd/server/start/start_master.go

```diff
@@ -378,7 +380,7 @@ func (m *Master) Start() error {
 	}
 
 	if m.api {
-		glog.Infof("Starting master on %s (%s)", m.config.ServingInfo.BindAddress, version.Get().String())
+		glog.Infof(MasterStartupMessage, m.config.ServingInfo.BindAddress, version.Get().String())
```

Comment:
Pretty sure `go vet` will catch it
https://code.google.com/p/go/source/detail?r=29665c4bae69
(they should of course be constants not vars)

But I agree it's awkward to impose this on the various places these
messages originate. OTOH... it's an extra signal to pay attention when
making changes to that log message.

On Tue, Sep 1, 2015 at 11:09 PM, Jordan Liggitt <notifications@github.com>
wrote:

> In pkg/cmd/server/start/start_master.go
> <https://github.com/sosiouxme/origin/pull/3#discussion_r38495568>:
>
> > @@ -378,7 +380,7 @@ func (m *Master) Start() error {
> >  	}
> >
> >  	if m.api {
> > -		glog.Infof("Starting master on %s (%s)", m.config.ServingInfo.BindAddress, version.Get().String())
> > +		glog.Infof(MasterStartupMessage, m.config.ServingInfo.BindAddress, version.Get().String())
>
> hrmm, it makes it more painful to actually use the message and see the
> format string is being used correctly... not sure if go vet will catch
> format errors in referenced format strings like this
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sosiouxme/origin/pull/3/files#r38495568>.
>